mtags-shared/src/main/scala/scala/meta/internal/pc/AutoImportPosition.scala

```diff
@@ -26,11 +28,17 @@ object AutoImportPosition {
   // class Main {
   //   def foo<COMPLETE> // inferred indent is 2 spaces.
   // }
+  @tailrec
   def inferIndent(lineStart: Int, text: String): Int = {
     var i = 0
     while (lineStart + i < text.length && text.charAt(lineStart + i) == ' ') {
       i += 1
     }
-    i
+
+    val pos = lineStart + i
+    if (pos < text.length() && text.charAt(pos) == '\n') {
```

Comment:
Just to double check: are we sure that just `\n` is a safe choice that will cover all the cases (for example LF vs CRLF encodings?). Would something like `System.lineSeparator()` be a better choice? :slightly_smiling_face:

Reply:
Right, always forget about it, maybe we should make sure this works when line ends with `\r\n`

Reply:
I'll fix it and add an lsp test to make sure it works on windows. Thanks for with the reviews @filipwiech, your contribution is much appreciated. 😃